assignments/callbacks.js

```diff
@@ -27,12 +27,22 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, function (length) {
+  console.log(length);
+});
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length - 1]);
 }
 
+last(items, function (last) {
+  console.log(last);
+})
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
 }
```

Comment:
check out the MDN for help on the contains problem. There is a method called includes that could help here.